### PR TITLE
issue_690: Validation for attachments in report form

### PR DESCRIPTION
**Pull request by arslanbekova**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,7 +10,8 @@
     "build:devenv": "webpack --mode production --env devenv",
     "patch": "npx mversion patch",
     "test": "jest",
-    "test:watch": "jest --watch"
+    "test:watch": "jest --watch",
+    "lint": "lint-staged"
```
Comment (MariiaNebesnova) [inline package.json:14]: Looks like you've run this command, but it ignored configuration of .eslintrc.json. Unfortunately, it leaded to all these changes

File: src/features/reportProblem/ReportProblem.tsx [hunk 2]
```diff
@@ -68,44 +78,50 @@ export const ReportProblem = () => {
...
+    // request
+    //     .get(HttpEndpoint.PING_SMTP)
+    //     .then((response) => {
+    //       if (response === 1) {
+    //         setServerPingInProcess(false);
+    setIsModalOpen(true);
+    //   } else showExceprionConfirm();
+    // })
+    // .catch(() => showExceprionConfirm())
+    // .finally(() => setServerPingInProcess(false));
```
Comment (MariiaNebesnova) [inline src/features/reportProblem/ReportProblem.tsx:110]: please, don't forget to uncomment it

File: src/features/reportProblem/ReportProblem.tsx [hunk 3]
```diff
@@ -190,33 +216,80 @@ export const ReportProblem = () => {
...
+                placement="right"
+                title={"Only 10 files can be uploaded"}
+                trigger={
+                  fileList.length >= MAX_COUNT_FILES ? ["hover", "focus"] : ""
                 }
               >
-                Upload
-              </Button>
+                <Button
+                  disabled={fileList.length >= MAX_COUNT_FILES}
```
Comment (MariiaNebesnova) [inline src/features/reportProblem/ReportProblem.tsx:283]: how does this button is disabled when `MAX_FILES_SIZE_MB` exceed?

File: src/features/reportProblem/ReportProblem.tsx [hunk 4]
```diff
@@ -190,33 +216,80 @@ export const ReportProblem = () => {
...
-              <Button
-                icon={
-                  <span role="img" className="anticon anticon-upload">
-                    <UploadSimple />
-                  </span>
+            <Upload
+              name="attachments"
+              onChange={handleUploadChange}
+              {...{ fileList }}
+              multiple={true}
```
Comment (MariiaNebesnova) [inline src/features/reportProblem/ReportProblem.tsx:273]: I believe `={true}` can be omitted

File: src/features/reportProblem/ReportProblem.tsx [hunk 5]
```diff
@@ -190,33 +216,80 @@ export const ReportProblem = () => {
...
+            >
+              <Tooltip
+                placement="right"
+                title={"Only 10 files can be uploaded"}
+                trigger={
+                  fileList.length >= MAX_COUNT_FILES ? ["hover", "focus"] : ""
```
Comment (MariiaNebesnova) [inline src/features/reportProblem/ReportProblem.tsx:279]: And what about `MAX_FILES_SIZE_MB `? will we notify user about it somehow?

File: src/features/reportProblem/ReportProblem.tsx (hunk 3 above)
Comment (arslanbekova) [reply src/features/reportProblem/ReportProblem.tsx:283]: I am not sure that we should disable upload button when MAX_FILES_SIZE_MB exceed. I think this case is missed in requirements.
I can clarify it with Semyon or with someone else

File: src/features/reportProblem/ReportProblem.tsx (hunk 5 above)
Comment (arslanbekova) [reply src/features/reportProblem/ReportProblem.tsx:279]: Now we notify user with error message on 255 row, but we can also add extra tooltip on upload button. This case is also missed and should be clarify

**Pull request merged**